server/routes/api/v1/petsRouter.js

```diff
@@ -23,7 +37,7 @@ petsRouter.get("/type", async (req, res) => {
   }
 })
 
-petsRouter.get("/type/:id", async (req, res) => {
+petsRouter.get("/:type/:id", async (req, res) => {

```

Comment:
Be careful with double dynamic endpoints - things can get messy very quickly. If this is functioning as expected, then you can leave it, but in the future we want to be cautious against having too much dynamically happening in our routes